prometheus-mixin/Makefile

```diff
@@ -0,0 +1,56 @@
+# Prometheus Mixin Makefile
```

Comment:
let's put `prometheus-mixin` in a `contrib` top level directory

Reply:
also please add a README.md file in this directory explaining what it is

Reply:
Sure, updated.